<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_about</class>
 <widget class="QDialog" name="dlg_about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Expense Tracker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_title">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Expense Tracker</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_version">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Version </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_author">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>(c) Frank Eisele, 2024</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Changelog:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="txt_browser"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
